examples/clinical_pipeline/clinical_processing_pipeline.py

```diff
@@ -1,48 +1,153 @@
 import sys
+import json
+import yaml
 import time
 
-import yaml
+from fortex.spacy import SpacyProcessor
+
 from mimic3_note_reader import Mimic3DischargeNoteReader
+from utterance_searcher import LastUtteranceSearcher
+from stave_backend.lib.stave_session import StaveSession
 
+from forte.data.readers import RawDataDeserializeReader
 from forte.common.configuration import Config
 from forte.data.data_pack import DataPack
 from forte.pipeline import Pipeline
+from fortex.elastic import ElasticSearchPackIndexProcessor
 from forte.processors.writers import PackIdJsonPackWriter
 
-from fortex.elastic import ElasticSearchPackIndexProcessor
-from fortex.huggingface.bio_ner_predictor import BioBERTNERPredictor
-from fortex.huggingface.transformers_processor import BERTTokenizer
+from forte.processors.writers import PackIdJsonPackWriter
+from ft.onto.base_ontology import Sentence, EntityMention
+
 from fortex.nltk import NLTKSentenceSegmenter
 
-def main(input_path: str, output_path: str, max_packs: int = -1):
-    pl = Pipeline[DataPack]()
-    pl.set_reader(
-        Mimic3DischargeNoteReader(), config={"max_num_notes": max_packs}
-    )
-    pl.add(NLTKSentenceSegmenter())
+from ftx.medical.clinical_ontology import (
+    NegationContext,
+    MedicalEntityMention,
+    MedicalArticle,
+)
+from fortex.health.processors.negation_context_analyzer import (
+    NegationContextAnalyzer,
+)
+from fortex.health.processors.icd_coding_processor import ICDCodingProcessor
+
+
+def get_json(path: str):
+    file_obj = open(path)
+    data = json.load(file_obj)
+    file_obj.close()
+    return data
+
+
+def update_stave_db(
+    default_project_json, chat_project_json, chat_doc_json, config
+):
+    project_id_base = 0
+    with StaveSession(url=config.Stave.url) as session:
+        session.login(username=config.Stave.username, password=config.Stave.pw)
+
+        projects = session.get_project_list().json()
+        project_names = [project["name"] for project in projects]
+
+        if (
+            default_project_json["name"] in project_names
+            and chat_project_json["name"] in project_names
+        ):
+
+            base_project = [
+                proj
+                for proj in projects
+                if proj["name"] == default_project_json["name"]
+            ][0]
+            return base_project["id"]
 
-    config = yaml.safe_load(open("bio_ner_config.yml", "r"))
+        resp1 = session.create_project(default_project_json)
+        project_id_base = json.loads(resp1.text)["id"]
+
+        resp2 = session.create_project(chat_project_json)
+        project_id_chat = json.loads(resp2.text)["id"]
+
+        chat_doc_json["project_id"] = project_id_chat
+        doc_id = session.create_document(chat_doc_json)
+        project_list = session.get_project_list().json()
+
+    return project_id_base
+
+
+def main(
+    input_path: str, output_path: str, max_packs: int = -1, run_ner_pipeline=0
+):
```

Comment:
If I ran this I will have `ModuleNotFoundError: No module named 'ftx.medical`, instructions to add the ontology?